src/coin-selection/lib/Convex/CoinSelection.hs

```diff
@@ -542,11 +543,11 @@ balanceTx ::
   -- | The balanced transaction body and the balance changes (per address)
   m (C.BalancedTxBody ERA, BalanceChanges)
 balanceTx dbg returnUTxO0 walletUtxo txb changePosition = do
-  params <- queryProtocolParameters
+  params <- Eras.babbageProtocolParams <$> queryProtocolParameters
```

Comment:
So this is still `BabbageEra`. When do we change this to `Conway`?

Reply:
In a separate PR, but this can be done after the hardfork. It is still possible to submit babbage-era transactions in conway